tools/test/CMakeLists.txt

```diff
@@ -50,3 +50,35 @@ iree_lit_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+if((IREE_TARGET_BACKEND_LLVM_CPU OR DEFINED IREE_HOST_BINARY_ROOT) AND
+   IREE_HAL_DRIVER_LOCAL_SYNC)
+  iree_bytecode_module(
+    NAME
+      iree_run_module_bytecode_module_llvm_cpu
+    SRC
+      "iree-run-module.mlir"
+    MODULE_FILE_NAME
+      "iree_run_module_bytecode_module_llvm_cpu.vmfb"
+    C_IDENTIFIER
+      "iree_tools_test_iree_run_module_bytecode_module_llvm_cpu"
+    FLAGS
+      "--iree-hal-target-backends=llvm-cpu"
+  )
```

Comment:
Can you switch this to use VMVX instead? (always enabled in the compiler, more portable)

Reply:
Done.